### ISSUE TITLE
Arabic Keyboard Layout should allow picking which type of numbers to use

### ISSUE THREAD

**Issue by maddada** [futo-org/futo-keyboard-layouts#128]
### Language

Arabic

### Description

Version is 0.124 (I'm in the play store beta) 

The arabic layout in many countries such as UAE uses the "[Western Arabic Numerals](https://en.m.wikipedia.org/wiki/Arabic_numerals)": 0 1 2 3 4 5 6 7 8 9

But FUTO keyboard's Arabic layout shows the "Eastern Arabic Numerals" for the numbers row as you can see in the screenshot

![Screenshot_20241201_000401_com_microsoft_emmx_ChromeTabbedActivity](https://github.com/user-attachments/assets/eb2415dd-36bf-4c4a-814e-f56ec06e872f)

Please allow us to switch the numeral system in the Arabic layout (the long press can still be the other type)

Thank you for all the great work. 

### Complexity

No

**Comment by tenextractor** [comment]
@maddada Can you specify which countries use Western Arabic numerals and which use Eastern Arabic numerals?

**Comment by maddada** [comment]
Hey @tenextractor,

UAE, Libya, Tunisia, Morocco, Algeria use 1 2 3
(Western Arabic Numerals)

Saudi Arabia, Oman, Syria, Yemen, Egypt use ١  ٢  ٣
(Eastern Arabic Numerals)

Some countries use both so it would be awesome if we were able to set 1 of them as default if possible.

Thank you!

**Comment by tenextractor** [comment]
Thanks. I am planning on making it locale specific, which means for example Arabic (UAE) would have Western Arabic numbers and Arabic (Egypt) would have Eastern Arabic numbers. Is this a good idea?

**Comment by Dhia-zorai** [comment]
Yes please,  i recently switched to futo and this is the only issue i have, we only use western arabic numerals in north africa.

**Comment by Dhia-zorai** [comment]
> Thanks. I am planning on making it locale specific, which means for example Arabic (UAE) would have Western Arabic numbers and Arabic (Egypt) would have Eastern Arabic numbers. Is this a goo

> Thanks. I am planning on making it locale specific, which means for example Arabic (UAE) would have Western Arabic numbers and Arabic (Egypt) would have Eastern Arabic numbers. Is this a good idea?

Yes please, i recently switched to Futo and this is the only issue i have, we only use western arabic numerals in north africa, so having two options would be great.
